udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +90,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
    */
+  private int getCommentCount(HttpServletRequest request) {
+    // Get the input from the form.
+    String commentCountString = request.getParameter("comment-count");
+
+    // Convert the input to an int.
+    int commentChoice;
+    try {
+      commentChoice = Integer.parseInt(commentCountString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + commentCountString);
+      return -1;
+    }
 
-  private String getRequestParameter(HttpServletRequest request, String comment, String defaultValue) {
+    return commentChoice;
```

Comment:
What should be returned if it's 0. Is that a reasonable return value?

Reply:
The site will only let numbers 1 through 50 go through, error checker is there for safe practice.